record-and-playback/core/lib/recordandplayback/edl/video.rb

```
@@ -685,10 +686,11 @@ def self.composite_cut(output, cut, layout, videoinfo)
             ffmpeg_filter << "setpts=PTS*#{scale}," unless scale.nil?
             # Apply PTS offset so '0' time is aligned
             ffmpeg_filter << "setpts=PTS-#{ms_to_s(in_time)}/TB,"
-            # Clean up the framerate
-            ffmpeg_filter << "fps=#{layout[:framerate]},"
             # Extend the video if needed
-            ffmpeg_filter << 'tpad=stop=-1:stop_mode=clone,'
+            ffmpeg_filter <<  "color=c=white:s=#{tile_width}x#{tile_height}:r=#{layout[:framerate]}[#{pad_name}_tpad];"
```

RuboCop (1.73)

[convention] Operator `<<` should be surrounded by a single space.

(Layout/SpaceAroundOperators)